ansible/files/servername.conf

```diff
@@ -0,0 +1 @@
+ServerName tbg
```

Comment:
I suggest suffixing this with dev, such as `tbg-dev` instead

Reply:
Ok, fixed - the server name has been set to `tbg-dev` throughout. I did change the first commit, just adding OpenPGP signature to it.

Reply:
Oh, the wiki page at https://issues.thebuggenie.com/wiki/TheBugGenie%3ADevelopment%3AVagrant  has been updated too (kinda in advance).